pvr.stalker/resources/language/resource.language.en_us/strings.po

```diff
@@ -29,8 +29,8 @@ msgid "Misc"
 msgstr "Misc"
 
 msgctxt "#30003"
-msgid "Connection Timeout"
-msgstr "Connection Timeout"
+msgid "Connection Timeout (seconds)"
+msgstr "Connection Timeout (seconds)"
```

Comment:
Drop all changes to string files other than en_gb. Translators will take care of them.